src/vector/platform/ElectronPlatform.tsx

```diff
@@ -57,6 +57,8 @@ interface SquirrelUpdate {
     updateURL: string;
 }
 
+const LEGACY_PROTOCOL = "element";
+const OIDC_PROTOCOL = "io.element.desktop";
```

Comment:
What is the purpose of making these symbolic constants instead of inlining them? I don't find it particularly helpful in terms of readability: it means I now have to look at two completely different parts of the file to understand what the code is doing. And they only seem to be used in one place.

It would be better if the constants were documented.

Reply:
Happy to inline them, was aligning with the electron side of the change in the hopes that in the future they can share a const

Reply:
as long as they don't share a const, I think they're better inline. If they *could* share a const, it would need documenting :)

Suggest you inline them, thanks